app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -28,7 +28,7 @@ keywords:
 ---
 
 
-You need a bachelor's degree to teach in primary, secondary and special schools in England. This does not have to be a bachelor's degree in teaching.
+You need a bachelor's degree in any subject to teach in primary, secondary and special schools in England.
```

Comment:
@EllieNodder Do we also need to add the sentence you have on the if you don't have a degree page about providers wanting to make sure you have a good subject knowledge for secondary?